phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java

```diff
@@ -342,6 +342,7 @@
     public static final long DEFAULT_GLOBAL_INDEX_ROW_AGE_THRESHOLD_TO_DELETE_MS = 7*24*60*60*1000; /* 7 days */
     public static final boolean DEFAULT_INDEX_REGION_OBSERVER_ENABLED = true;
     public static final long DEFAULT_INDEX_REBUILD_PAGE_SIZE_IN_ROWS = 32*1024;
+    public static final long DEFAULT_UNGROUPED_AGGREGATE_PAGE_SIZE_IN_ROWS = 32*1024;
```

Comment:
How did we arrive at 32768 rows? Since we aren't using a notion of row size for paging, but rather absolute "number of rows", can we replace the 1024 with say 1000 just to avoid confusion (one may think this is 32KB total row size)?

Reply:
No specific reason, I guess I like powers of 2. I will change it to 32*1000. Based on my testing on a real cluster, 32K gives very good performance without timeouts.